debezium-connect-rest-extension/pom.xml

```diff
@@ -75,10 +79,21 @@
         <filtering>true</filtering>
         <directory>src/main/resources</directory>
         <includes>
-          <include>**/META-INF/services/*</include>
+          <include>*</include>
```

Comment:
From the Zen of Python: Explicit is better than implicit? @jpechane  @ani-sha  should we maybe define the previous include and add the `build.version` explicitly here? WDYT?

Reply:
@rk3rn3r No strong opinion from my side so if you feel it is better then I am fine with it